lib/models/yust_notification.dart

```diff
@@ -6,8 +6,7 @@ part 'yust_notification.g.dart';
 
 @JsonSerializable(anyMap: true)
 class YustNotification extends YustDoc {
-
-  static final setup = YustDocSetup(
+  static final setup = YustDocSetup<YustNotification>(
```

Comment:
Specifying the `<YustNotification>` part is in most cases not necessary, it  is only intended to make it more explicit.